app/views/shared/_navbar.html.erb

```diff
@@ -20,11 +15,11 @@
           <li class="nav-item">
             <%= link_to "Profile", current_user, class: "nav-link" %>
           </li>
-          <li class="nav-item" >
+          <li class="nav-item" style="font-size: 1.05em">
             <%= link_to "log_out", destroy_user_session_path, method: :delete, class: "nav-link border-start button-square-right logout ms-3" %>
           </li>
         <% else %>
```

Comment:
This doesn't need an opening if statement? 
Or does this else work differently?

Reply:
It is there in line 14/ line 19.

Reply:
Ah sorry, I didn't expand the view